README.md

```diff
@@ -1,8 +1,108 @@
 # dcos-images
 
-The reference for DC/OS images and their prerequisites
-[DC/OS Platform Version Compatibility Matrix](https://docs.mesosphere.com/version-policy/#dcos-platform-version-compatibility-matrix)
+The reference for DC/OS images and how to build them
 
-Each dcos_images file should be a mapping of docker_version to region_ids to image_ids
+## Which DC/OS image should I use?
+Look for the dcos_images.yaml file in the directory path that matches your requirements.  
+Example: [centos/7.4/aws/DCOS-1.11.3/docker-1.13.1/dcos_images.yaml](https://github.com/dcos/dcos-images/blob/master/centos/7.4/aws/DCOS-1.11.3/docker-1.13.1/dcos_images.yaml).
 
-(work in progress)
+## How can I see a high-level view of what we support?
+1. ```git clone https://github.com/dcos/dcos-images.git ~/dcos-images```
+2. ```brew install tree``` or ```apt install tree```
+3. ```cd ~/dcos-images```
+4. ```tree```
+
+There's also a [DC/OS Platform Version Compatibility Matrix](https://docs.mesosphere.com/version-policy/#dcos-platform-version-compatibility-matrix)
+
```

Comment:
Please add a very high level description of this project.

It should answer - How to can I add a new version of docker, a new pre-requisite and quality it for DC/OS.